davidl-max/starter-code/scripts/article.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var articles = [];
+
+function Article (rawDataObj) {
+  this.author = rawDataObj.author;
+  this.authorUrl = rawDataObj.authorUrl;
+  this.title = rawDataObj.title;
+  this.category = rawDataObj.category;
+  this.body = rawDataObj.body;
+  this.publishedOn = rawDataObj.publishedOn;
+}
+
+Article.prototype.toHtml = function() {
+  var $newArticle = $('article.template').clone();
+  $newArticle.removeClass('template');
+  if (!this.publishedOn) {
+    $newArticle.addClass('draft');
+  }
+  $newArticle.attr('data-category', this.category);
+  $newArticle.attr('data-attribute', this.author);
```

Comment:
You should store the author's name in `data-author` to indicate that it is the author of the article, rather than the more generic name `data-attribute`.